mmdeploy/utils/constants.py

```diff
@@ -24,6 +24,7 @@ class Task(AdvancedEnum):
     CLASSIFICATION = 'Classification'
     OBJECT_DETECTION = 'ObjectDetection'
     INSTANCE_SEGMENTATION = 'InstanceSegmentation'
+    POSE_DETECTION = 'PoseDetection'
```

Comment:
Should we define different task for 2d/3d kpt?

Reply:
For now, we did not fully compare the difference of 2d and 3d kpt models. Don't know if it's necessary.